### PR TITLE
EZP-30885: Introduced strict types for LanguageService

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-30885](https://jira.ez.no/browse/EZP-30885)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `master`
| **BC breaks**      | yes!
| **Tests pass**     | yes
| **Doc needed**     | no

This PR introduces strict types for the `\eZ\Publish\API\Repository\LanguageService`

**TODO**:
- [X] Implement feature / fix a bug.
- [X] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

**QA**
- [x] Sanities for language management
- [x] Sanities for COTF
- [x] Sanities for creating content
- [x] Sanities for creating content translation
- [x] Sanities for UDW
- [x] Sanities for language lists

### CODE REVIEW

File: eZ/Publish/SPI/Repository/Tests/Decorator/LanguageServiceDecoratorTest.php [hunk 1]
```diff
@@ -108,7 +108,7 @@ public function testLoadLanguageByIdDecorator()
         $serviceMock = $this->createServiceMock();
         $decoratedService = $this->createDecorator($serviceMock);
 
-        $parameters = ['random_value_5ced05ce0e4f86.21411523'];
+        $parameters = [100];
 
         $serviceMock->expects($this->once())->method('loadLanguageById')->with(...$parameters);
```
Comment (alongosz) [inline eZ/Publish/SPI/Repository/Tests/Decorator/LanguageServiceDecoratorTest.php]: You can probably pass `100` directly here, this code was probably generated in more or less automated way, no one said it should be the same for test :)

**Comment by micszo** [comment]
Discovered while testing: https://jira.ez.no/browse/EZP-31027.

**Comment by micszo** [comment]
Discovered while testing: https://jira.ez.no/browse/EZP-31033

**Comment by micszo** [comment]
Discovered while testing: https://jira.ez.no/browse/EZP-31034

**Pull request merged**